tripaldocker/Dockerfile-php8.2-pgsql13

```diff
@@ -213,7 +213,6 @@ RUN mv /app/tripaldocker/init_scripts/supervisord.conf /etc/supervisord.conf \
 ## Make global commands.
 RUN ln -s /var/www/drupal/vendor/phpunit/phpunit/phpunit /usr/local/bin/ \
   && ln -s /var/www/drupal/vendor/drush/drush/drush /usr/local/bin/ \
-  && ln -s /var/www/drupal /var/www/drupal9
```

Comment:
This is here for backwards compatibility with old tripaldockers. The main directory is now not specific to verions (i.e. `/var/www/drupal`) so this line just adds a symbolic link for backwards compatiblity with previous images. We would like to keep it in place for a bit longer.